fideslog/api/api/main.py

```diff
@@ -2,6 +2,8 @@
 from datetime import datetime
 from http import HTTPStatus
 from typing import Callable
+import sys
+sys.path.append(".")
```

Comment:
This is a hack to make module imports work. Without it the the server will through errors like this
```
fideslog-fideslog-1  |     from routes.api import api_router
fideslog-fideslog-1  |   File "/fideslog/api/routes/api.py", line 5, in <module>
fideslog-fideslog-1  |     from ..config import config
fideslog-fideslog-1  | ImportError: attempted relative import beyond top-level package
```